doc/releases/changelog-0.26.0.md

```diff
@@ -4,27 +4,84 @@
 
 <h3>New features since last release</h3>
 
-<h4>QNSPSA optimizer 😤</h4>
+<h4>Classical shadows 👤</h4>
 
-* A new optimizer called `qml.QNSPSAOptimizer` is available that implements the quantum natural simultaneous
-  perturbation stochastic approximation (QNSPSA) method based on 
-  [Simultaneous Perturbation Stochastic Approximation of the Quantum Fisher Information](https://quantum-journal.org/papers/q-2021-10-20-567/). 
-  [(#2818)](https://github.com/PennyLaneAI/pennylane/pull/2818) 
+* All-new features for implementing the classical-shadows protocol are now available. 
+  [(#2820)](https://github.com/PennyLaneAI/pennylane/pull/2820)
+  [(#2821)](https://github.com/PennyLaneAI/pennylane/pull/2821)
+  [(#2871)](https://github.com/PennyLaneAI/pennylane/pull/2871)
+  [(#2968)](https://github.com/PennyLaneAI/pennylane/pull/2968)
+  [(#2959)](https://github.com/PennyLaneAI/pennylane/pull/2959)
+  [(#2968)](https://github.com/PennyLaneAI/pennylane/pull/2968)
 
-  `qml.QNSPSAOptimizer` can be viewed as a second-order SPSA algorithm. It requires 10 circuit 
-  executions per optimization step compared to 3 for `qml.SPSAOptimizer`.
-  The additional circuit executions are used to provide a stochastic estimation of a second-order
-  metric tensor, which often helps the optimizer to achieve faster convergence. 
+  The classical-shadow measurement protocol is described in detail in the
+  [classical shadows paper](https://arxiv.org/abs/2002.08953).
+  As part of the support for classical shadows in this release, two new finite-shot and fully-differentiable measurements are available: 
 
-  Use `qml.QNSPSAOptimizer` like you would any other optimizer:
+  - QNodes returning `qml.classical_shadow` will return two entities: 
+
+    + `bits`: 0 or 1 if the 1 or -1 eigenvalue is sampled, respectively
+    + `recipes`: the randomized Pauli measurements that are performed for each qubit, identified as a unique integer: 0 for Pauli X, 1 for Pauli Y, and 2 for Pauli Z.
```

Comment:
```suggestion
- QNodes returning the new measurement `qml.classical_shadow()` will return two entities;
  `bits` (0 or 1 if the 1 or -1 eigenvalue is sampled, respectively) and `recipes`
  (the randomized Pauli measurements that are performed for each qubit, labelled by integer):
```

Optional, but a bit more concise!

Reply:
I prefer this solely because for some reason `recipes` isn't rendered as code upon deployment 🤔